framework/common/WindowContext.kt

```diff
@@ -29,20 +29,20 @@ sealed interface WindowContext {
     class Test(override var height: Int, override var width: Int, override var x: Int, override var y: Int): WindowContext
 
     class Compose(val window: ComposeWindow): WindowContext {
-        override var height: Int
+        override var height
             get() = window.size.height
-            set(value) { value }
+            set(value) { height = value }
```

Comment:
We don't use `set` at all, but the compiler was complaining and we may as well set it.